### PR TITLE
Added tbench test - v2

### PR DESCRIPTION
**Pull request by poojabs7**
Added tbench test that produces only the TCP and process load.

Signed-off-by: poojabs <pooja@oc2013156016.ibm.com>

### CODE REVIEW

File: perf/tbench.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        # should support separate I suppose, but nobody uses it
+        nprocs = self.params.get('nprocs', default=None)
+        args = self.params.get('args',  default=None)
+        if not nprocs:
+            nprocs = commands.getoutput("nproc")
+        args = args + ' %s' % nprocs
+
+        pid = os.fork()
+        if pid:                         # parent
+            client = self.srcdir + '/client.txt'
```
Comment (PraveenPenguin) [inline perf/tbench.py]: can you please use `os.path.join` ?

File: perf/tbench.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+            args = '-c ' + client + ' ' + '%s' % args
```
Comment (PraveenPenguin) [inline perf/tbench.py]: string add not looks good using `+`

it can be like 
`args = '-c %s %s'  % (client,args)`

File: perf/tbench.py [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+            cmd = os.path.join(self.srcdir, "tbench") + " " + args
+            # Standard output is verbose and merely makes our debug logs huge
+            # so we don't retain it.  It gets parsed for the results.
+            self.results = process.system_output(cmd, shell=True)
+            os.kill(pid, signal.SIGTERM)    # clean up the server
+        else:                           # child
+            server = self.srcdir + '/tbench_srv'
```
Comment (PraveenPenguin) [inline perf/tbench.py]: here also please use `os.path.join`

**Comment by poojabs7** [comment]
Raised new PR 
https://github.com/avocado-framework-tests/avocado-misc-tests/pull/156

**Pull request closed without merging**